requirements/edx/github.in

```diff
@@ -17,7 +17,7 @@
 #
 # For example:
 #
-#   git+https://github.com/edx/edx-lint.git@v0.3.2#egg=edx_lint==0.3.2
```

Comment:
@sarina I've got a [PR in the works](https://github.com/openedx/edx-platform/pull/30719) that will update all the links in this file and regenerate the `.txt` files, plus some other changes. To spare me the merge conflicts do you mind removing the changes to `./requirements` from this PR?

Reply:
It would probably be easier for me to close the PR, and regenerate it when you're done.

Reply:
Nevermind then, I would rather handle the merge conflicts than review this PR again 😅

Reply:
oh, just remove anything in the requirements dir? that's easy

Reply:
awesome, thanks 👍🏻

Reply:
done! now to wait and see how many tests I need to fix 🤪